cmd/deploy.go

```diff
@@ -152,16 +144,30 @@ var deployCmd = &cobra.Command{
 		for _, node := range c.Nodes {
 			go func(node nodes.Node) {
 				defer wg.Done()
-				err := node.PostDeploy(ctx, c.Runtime, c.Nodes)
+				err := node.PostDeploy(ctx, c.Nodes)
 				if err != nil {
 					log.Errorf("failed to run postdeploy task for node %s: %v", node.Config().ShortName, err)
 				}
 			}(node)
 		}
 		wg.Wait()
 
+		// Update containers after postDeploy action
+		containers, err = c.ListContainers(ctx, labels)
+		if err != nil {
+			return err
+		}
+
+		// generate graph of the lab topology
+		if graph {
+			if err = c.GenerateGraph(topo); err != nil {
+				log.Error(err)
+			}
+		}
+		log.Debug("containers created, retrieving state and IP addresses...")
```

Comment:
this log message seems to have been moved not on purpose

Reply:
yep, will fix